examples/static_set/host_bulk_perfect_example.cu

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Unlike in the benchmark code, for example code, I like the idea of having this copy of the file rather than use templates.  Templates are harder to read and examples should be kept simple.  So I'm cool with this file, even though it looks very similar to the other `host_bulk_example.cu`.